homework-submission/React-Week3/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './todoapp/Header';
+import TodoList from './todoapp/TodoList';
+import Photo from './todoapp/Picture';
+import logo from './logo.svg';
+import Form from './todoapp/Form';
+import todos from './todos.json';
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: todos,
+    };
+  }
+  addTodoItem = (e) => {
+    e.preventDefault();
+    const newTodo = {
+      "id": this.state.todos.length + 1,
```

Comment:
the Idea of `id` is to set a unique property for the object!
So the length in sommepoint won't be unique think about this:
>you have 3 items the length is 3 ids: 1, 2, 3
>you deleted a one in the middle so your ids now: 1, 3
>you added a new one your ids: 1, 3, 3

see? 🙃